app/views/editor/podcast/PodcastSearchView.js

```diff
@@ -0,0 +1,14 @@
+const SearchView = require('views/common/SearchView')
+
+class PodcastSearchView extends SearchView {
+  id = 'editor-podcast-home-view'
+  modelLabel = 'Podcast'
+  model = require('models/Podcast')
+  modelURL = '/db/podcast'
+  tableTemplate = require('app/templates/editor/course/table')
```

Comment:
I think it will be weird to reuse the course editor table, and we should start a new table template for podcasts (we will probably want to have stuff like `guestImage` and `uploadDate` displayed in the table, too).